src/multi_vector_simulator/utils/constants.py

```diff
@@ -88,7 +88,7 @@
 # list of csv filename which must be present within the CSV_ELEMENTS folder with the parameters
 # associated to each of these filenames
 REQUIRED_CSV_PARAMETERS = {
-    CONSTRAINTS: [MINIMAL_RENEWABLE_FACTOR],
+    CONSTRAINTS: [MINIMAL_RENEWABLE_FACTOR, MAXIMUM_EMISSIONS],
```

Comment:
I would have thought that we keep the ´MAXIMUM_EMISSIONS` as extra parameters with default values? Why not?

Reply:
Every required parameters listed in `KNOWN_EXTRA_PARAMETERS` will have the default value if not provided